lynx/src/main/res/layout/lynx_view.xml

```diff
@@ -41,23 +41,35 @@
         android:layout_centerVertical="true" android:layout_alignParentRight="true"
         android:layout_alignParentEnd="true"/>
 
-    <!-- Filter EditText -->
+    <!-- Filter Layout -->
 
-    <EditText
-        android:id="@+id/et_filter"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:padding="@dimen/filter_edit_text_padding"
-        tools:text="Filter"
-        android:background="@color/black"
-        android:textColor="@color/white"
-        android:textColorHint="@color/gray"
-        android:hint="@string/et_filter_hint_text"
-        android:layout_alignParentBottom="true"
-        android:layout_toLeftOf="@id/ib_share"
-        android:layout_centerVertical="true"/>
+      <LinearLayout
+          android:orientation="horizontal"
+          android:layout_alignParentBottom="true"
+          android:layout_centerVertical="true"
+          android:layout_toLeftOf="@id/ib_share"
+          android:layout_width="match_parent"
+          android:layout_height="match_parent">
 
+          <EditText
+              android:id="@+id/et_filter"
+              android:layout_width="0dp"
+              android:layout_weight="1"
+              android:layout_height="match_parent"
+              android:background="@color/black"
+              android:hint="@string/et_filter_hint_text"
+              android:padding="@dimen/filter_edit_text_padding"
+              android:textColor="@color/white"
+              android:textColorHint="@color/gray"
+              tools:text="Filter"/>
 
+          <Spinner
+              android:id="@+id/sp_filter"
+              android:layout_width="0dp"
+              android:layout_weight="1"
```

Comment:
We should change the layout_weight of this spinner or update the hint message used in the EditText. What do you think if we use 70 % of the space for the edit text and 30 % for the spinner?